ChangeLog

```diff
@@ -1,3 +1,47 @@
+Wed, 26 Jul 2017 15:27:20 +0200 IvozProvider Team <dev@list-ivozprovider.irontec.com>
+
+    * IvozProvider 1.5 released
+
+    * Administation Portal
+        - Added register status in Terminals, Friends and Accounts lists
+        - Billable calls now display used Peering Contract
+        - Created companies has IP filtering by default
```

Comment:
I think you meant _"Created companies have external calls limited by default (2 in retail, 10 in vPBX)"_